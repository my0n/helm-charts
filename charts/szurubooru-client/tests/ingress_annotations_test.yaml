suite: ingress uses annotations
templates:
  - ingress.yaml
tests:
  - it: uses annotations
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    set:
      ingress.enabled: true
      ingress.annotations:
        a8r.io/description: A very nice ingress
    asserts:
      - equal:
          path: metadata.annotations.[a8r.io/description]
          value: A very nice ingress