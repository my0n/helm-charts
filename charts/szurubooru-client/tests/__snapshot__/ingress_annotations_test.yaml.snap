should match snapshot:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      annotations:
        a8r.io/description: A very nice pod
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: szurubooru-client
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: szurubooru-client-1.0.0
      name: RELEASE-NAME-szurubooru-client
    spec:
      rules:
      - host: chart-example.local
        http:
          paths:
          - backend:
              service:
                name: RELEASE-NAME-szurubooru-client
                port:
                  number: 80
            path: /
            pathType: ImplementationSpecific
